src/main/java/com/cleanroommc/groovyscript/compat/mods/thaumcraft/aspect/Aspect.java

```diff
@@ -14,7 +14,10 @@
 @RegistryDescription
 public class Aspect extends VirtualizedRegistry<thaumcraft.api.aspects.Aspect> {
 
-    @RecipeBuilderDescription(example = @Example(".tag('humor').chatColor(14013676).component(aspect('cognitio')).component('perditio').image(resource('thaumcraft:textures/aspects/humor.png'))"))
+    @RecipeBuilderDescription(example = {
+            @Example(".tag('clay').chatColor(14013676).image(resource('placeholdername:textures/items/clay_2.png'))"),
+            @Example(".tag('snack').chatColor(14013676).component(aspect('cognitio')).component('clay').image(resource('placeholdername:textures/items/snack.png'))")
```

Comment:
should these color ints be hex ints?

Reply:
good point, it would probably be more readable as hex codes - ill change those